metalearners/_utils.py

```diff
@@ -7,9 +7,11 @@
 from operator import le, lt
 from pathlib import Path
 
+import narwhals as nw
```

Comment:
Consider using stable version, we try to make some promises for that (See [Perfect backwards compatibility policy
](https://narwhals-dev.github.io/narwhals/backcompat/))

```suggestion
import narwhals.stable.v1 as nw
```

Reply:
This is great - adapted all of the imports!